docs/overview.rst

```diff
@@ -0,0 +1,213 @@
+       understand for newcomers.
+.. [#] Remote Storage allows "ls" on a folder, but items are not sorted or
+       paginated.
+.. [#] Kinto uses the same mechanisms as Remote storage for conflict handling.
+.. [#] Notifications support is currently in the work.
+.. [#] Remote Storage supports OAuth2.0 implicit grant flow.
+.. [#] Support for decentralised discovery
+       `is planned <https://github.com/Kinto/kinto/issues/125>`_ but not
+       implemented yet.
+.. [#] Remote Storage doesn't define any default implementation (as it is
+       a procol) but makes it easy to start with JavaScript and Node.js.
+
+You can also read `a longer explanation of our choices and motivations behind the
+creation of Kinto <http://www.servicedenuages.fr/en/generic-storage-ecosystem>`_
+on our blog.
+
+
+.. _FAQ:
+
+FAQ
+===
+
+- How does Kinto compares to CouchDB / Remote Storage?
+
+    Before starting to create yet another data storage service, we had a long
+    look to the existing solutions, to see if that would make sense to extend
+    the community effort rather than re-inventing the wheel.
+
+    It appeared that solutions we looked at weren't solving the problems we had,
+    especially regarding fine-grained permissions.
+
+    To see how Kinto compares to these solutions,
+    read :ref:`the comparison table <comparison>`.
+
+- Can I encrypt my data?
+
+    Kinto server stores any data you pass to it, be it encrypted or not.
+    We make it easy to use encryption in our Kinto.js client
+    `using transformers <http://kintojs.readthedocs.org/en/latest/api/#transformers>`_.
+
+- Is there a package for my Operating System?
+
+    Not at the moment. We want to make it very easy to integrate with existing
+    operating systems, and this item is on our priority list.
+
+    However, we are not there just yet. We are `already integrated with docker <https://hub.docker.com/r/kinto/kinto-server/>`
+    and :ref:`easy to install with pip <installation>`.
+
+- Why did you chose to use Python rather than X?
+
+    We know and love `Python <python.org>`_ for its simplicity and ease to
+    learn, so it was an obvious choice the development team. In addition, the
+    operational team at Mozilla has good recipes and a lot of knowledge about
+    how to deploy python.
+
+    However, the protocol and concepts behind Kinto don't rely on Python *per se*,
+    so it is possible to have other Kinto implementations using other languages.
+
+- Is it Web Scale?
+
+    YES™.
+
+- Can I store files inside Kinto?
+
+    No. At the moment, Kinto is meant to be used as a JSON storage service, and
+    differs with file storage solutions. We might add this in the future if
+    the use-case appears, but it is not on our radar so far.
+
+
+- What is Cliquet? What is the difference between Cliquet and Kinto ?
+
+    Kinto is a server built upon a toolkit named Cliquet. All of the reusable
+    parts have been factorised inside the toolkit, whereas what makes Kinto
```

Comment:
This... this is super confusing. Also, "factorised" isn't a word. :wink: 
Consider (simply):
> Cliquet is a toolkit for designing micro-services. Kinto is a server built using that toolkit.